pensieve/cli.py

```diff
@@ -26,6 +26,12 @@ def format_date(date):
     return datetime.combine(date, datetime.min.time()).replace(tzinfo=pytz.utc)
 
 
+def date_range(start_date, end_date):
+    """Generator for a range of dates."""
```

Comment:
Which range of dates? :) If it includes the end_date, it is unlike other Python range operations, so it would be good to be explicit about that.

Reply:
Something like `inclusive_date_range` maybe?

Reply:
I think adjusting the docstring is enough; the name seems fine.

Reply:
Cool, I also updated the docstring.